docs/Changelog.md

```diff
@@ -10,6 +10,7 @@
 * [#5280](https://github.com/TouK/nussknacker/pull/5280) Security improvement: Checking if user has access rights to fragment's Category for fragments served by definitions API
 * [#5303](https://github.com/TouK/nussknacker/pull/5303) Added `skipNodeResults` parameter to API endpoints that return scenario validation results
 * [#5323](https://github.com/TouK/nussknacker/pull/5323) Removed support for [the legacy categories configuration format](https://nussknacker.io/documentation/docs/1.12/installation_configuration_guide/DesignerConfiguration/#scenario-type-categories)
+* [#5266](https://github.com/TouK/nussknacker/pull/5266) Fixed order of access of methods and keys in records
```

Comment:
TBH it's hard to understand what's that. Are you able to write from the perspective of a user?

Reply:
I agree, @mslabek , you find a security leak, point it out here :) order is only a way to fix it

Reply:
I've added an extended explanation with an example. 

As to if this is a security problem - I can't get any meaningful vulnerability from this. I've tried a lot of different approaches to get arbitrary code execution or anything - can't get to classloader, runtime but that is probably part our configuration of spel parser and part of its internal implementation. But that implementation may change so I agree it's kind of a security risk.

Reply:
I've shortened the description to be actually readable for users

Reply:
yeah, this is it!